R/theme-defaults.R

```diff
@@ -553,13 +553,16 @@ theme_void <- function(base_size = 11, base_family = "",
                        header_family = NULL,
                        base_line_size = base_size / 22,
                        base_rect_size = base_size / 22,
-                       ink = "black", paper = "white") {
+                       ink = "black", paper = alpha(ink, 0)) {
   half_line <- base_size / 2
 
   # Only keep indispensable text: legend and plot titles
   t <- theme(
     line =               element_blank(),
-    rect =               element_blank(),
+    rect =               element_rect(
+                           fill = paper, colour = NA, linewidth = 0, linetype = 1,
+                           inherit.blank = FALSE
+                         ),
```

Comment:
Curious why you are not just setting `plot.background` to a non-blank element?

Reply:
You're right to wonder about this! Some test was protesting that we couldn't inherit from a blank root element. I then tried `plot.background = element_rect(..., inherit.blank = FALSE)` , but that gets overruled by `theme(..., complete = TRUE)`. I then tried using `+theme(...)` at the end of the function to force `inherit.blank = FALSE` without being bound to `complete = TRUE`, but this felt like a dirty solution. I've landed on  the current solution because at least it is straightforward.

Reply:
that makes sense (in its own little way)